src/backends/meta-egl.c

```diff
@@ -656,23 +656,24 @@ meta_egl_stream_consumer_output (MetaEgl          *egl,
   return TRUE;
 }
 
-gboolean
+EGLint
```

Comment:
I would prefer we keep the API style consistent, and use the GError to pass the error. I made a patch that makes the GError carry the EGL error here: https://github.com/jadahl/mutter/commit/0a9a9b6fca32dc636f74a7bdb51e52945992d437
Then on the call site, you'd do if (error->domain == META_EGL_ERROR && error->code == EGL_ERROR_IN_QUESTION) ...